sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -404,6 +425,7 @@ class JDBCSuite extends SparkFunSuite
       numPartitions = 0,
```

Comment:
Will fix this after your PR https://github.com/apache/spark/pull/15966 is merged